### PR TITLE
Add armor modifiers from DFA110

### PR DESCRIPTION
**Pull request by mfrench23**

### CODE REVIEW

File: Library/Dungeon Fantasy RPG/Dungeon Fantasy RPG Equipment Modifiers.eqm [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+					"type": "eqp_modifier",
+					"version": 1,
+					"id": "42a9c6fa-0243-4c1d-95c6-7c2f460bedc9",
+					"name": "Dragonhide (+1 DR)",
+					"reference": "DFA110",
+					"cost_type": "to_base_cost",
+					"cost": "+29 CF",
+					"weight_type": "to_base_weight",
+					"weight": "x1.25",
+					"notes": "Heavy leather; aditional +1 DR vs fire"
```
Comment (richardwilkes) [inline Library/Dungeon Fantasy RPG/Dungeon Fantasy RPG Equipment Modifiers.eqm:24]: Typo in the word "additional"

File: Library/Dungeon Fantasy RPG/Dungeon Fantasy RPG Equipment Modifiers.eqm [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+					"cost": "+29 CF",
+					"weight_type": "to_base_weight",
+					"weight": "x1/3",
+					"notes": "Any plate"
+				},
+				{
+					"type": "eqp_modifier",
+					"version": 1,
+					"id": "da7aab67-2adb-455c-8371-2ac173df1f2c",
+					"name": "Ornate +1",
```
Comment (richardwilkes) [inline Library/Dungeon Fantasy RPG/Dungeon Fantasy RPG Equipment Modifiers.eqm:127]: I've not read up on this, but doesn't this grant a reaction modifier of some sort? If so, a suitable feature granting that modifier should be added to this equipment modifier (and the other Ornates)

File: Library/Dungeon Fantasy RPG/Dungeon Fantasy RPG Equipment Modifiers.eqm [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+					"reference": "DFA110",
+					"cost_type": "to_base_cost",
+					"cost": "+2 CF",
+					"notes": "Any plate that covers body or more"
+				},
+				{
+					"type": "eqp_modifier",
+					"version": 1,
+					"id": "90d015a6-b379-4b63-ab4d-93b86e7f738d",
+					"name": "Thieves'",
```
Comment (richardwilkes) [inline Library/Dungeon Fantasy RPG/Dungeon Fantasy RPG Equipment Modifiers.eqm:168]: Similar to ornate, doesn't this grant some sort of skill modifier to stealth? Again, if so, that should be added as a feature to this entry.

**Review by richardwilkes — CHANGES_REQUESTED** [review]

File: Library/Dungeon Fantasy RPG/Dungeon Fantasy RPG Equipment Modifiers.eqm (hunk 3 above)
Comment (mfrench23) [reply Library/Dungeon Fantasy RPG/Dungeon Fantasy RPG Equipment Modifiers.eqm:189]: Added a note. It's pretty fiddly.

**Comment by richardwilkes** [comment]
Looks good now... however, it appears that there are a number of conflicts due to changes made by others in the same file -- a guess is that one or both of you changed the sorting. Unfortunately, this will have to be sorted out before it can be merged.

**Comment by richardwilkes** [comment]
Hi, @mfrench23.

Did you close this because plan on submitting a different PR due to the conflicts? Your changes looked good to me in general and I can help you resolve the conflicts if that is a problem.

- Rich

**Comment by mfrench23** [comment]
Exactly - I ran into some troubles with my workflow and concluded it was
best to start from a fresh branch. Thank you!

**Pull request closed without merging**